ios/MarkdownLayoutManager.mm

```diff
@@ -25,9 +27,13 @@ - (void)drawBackgroundForGlyphRange:(NSRange)glyphsToShow atPoint:(CGPoint)origi
       CGFloat y = paddingTop + rect.origin.y;
       CGFloat width = markdownUtils.markdownStyle.blockquoteBorderWidth;
       CGFloat height = rect.size.height;
-      CGRect lineRect = CGRectMake(x, y, width, height);
-      [markdownUtils.markdownStyle.blockquoteBorderColor setFill];
-      UIRectFill(lineRect);
+      CGFloat nestShift = paddingLeft + width + markdownUtils.markdownStyle.blockquoteMarginLeft;
```

Comment:
1. Let's rename the variable from `nestShift` to `shift` to be consistent with iOS.
2. We should also include `markdownUtils.markdownStyle.blockquotePaddingLeft` (blockquote padding) which is a different thing than `markdownUtils.backedTextInputView.textContainerInset.left` (text input padding). Let's assign all intermediate variables and change this line to:
```suggestion
      CGFloat shift = blockquoteMarginLeft + blockquoteBorderWidth + blockquotePaddingLeft;
```